### PR TITLE
PS-7343 Verify Documentation for encryption Features which are GA (8.0)

### PR DESCRIPTION
**Pull request by patrickbirch**
modified:   data-at-rest-encryption.rst
modified:   data-masking.rst
modified:   encrypting-binlogs.rst
modified:   encrypting-doublewrite-buffers.rst
modified:   encrypting-redo-log.rst
modified:   encrypting-system-tablespace.rst
modified:   encrypting-tables.rst
modified:   encrypting-tablespaces.rst
modified:   encrypting-temporary-files.rst
modified:   verifying-encryption.rst

### CODE REVIEW

**Review by kathy-williamson — APPROVED** [review]

File: doc/source/security/vault.rst [hunk 1]
```diff
@@ -1,4 +1,4 @@
-.. _vault:
+https://github.com/percona/percona-server/pull/3806.. _vault:
```
Comment (dutow) [inline doc/source/security/vault.rst]: Is this pull request link intentional here?

Comment (patrickbirch) [reply doc/source/security/vault.rst]: No. Miscopied. Fixed.

File: doc/source/security/encrypting-doublewrite-buffers.rst [hunk 2]
```diff
@@ -4,16 +4,18 @@
...
 
-|Percona Server| encrypts the ``doublewrite buffer`` with the respective
-tablespace keys. Only encrypted tablespace pages are written as encrypted in the
-doublewrite buffer. Unencrypted tablespace pages will be written as
-unencrypted.
+Currently, |Percona| encrypts the ``doublewrite buffer`` with the
+tablespace keys using :variable:`innodb_parallel_dblwr_encrypt`.
+Encrypted tablespace pages are written as encrypted in the
+doublewrite buffer. Unencrypted tablespace pages are not encrypted in the
+doublewrite buffer.
```
Comment (satya-bodapati) [inline doc/source/security/encrypting-doublewrite-buffers.rst]:
```suggestion
Currently, |Percona| encrypts the ``doublewrite buffer`` using :variable:`innodb_parallel_dblwr_encrypt`.
Encrypted tablespace pages are written as encrypted in the
doublewrite buffer using their respective tablespace keys.
Unencrypted tablespace pages are not encrypted in the
doublewrite buffer.
```

File: doc/source/security/encrypting-redo-log.rst [hunk 3]
```diff
@@ -16,16 +19,17 @@ tablespace encryption key.
...
-log data, by default, is 'OFF'.  
+Determines the encryption for redo log data for tables.
 
 When you enable `innodb_redo_log_encrypt` any existing redo log pages stay
-unencrypted, and new pages are encrypted when they are written to disk. If you
-disable `innodb_redo_log_encrypt`, any encrypted pages remain encrypted, but
+unencrypted, and new pages are encrypted when they are written to disk.
+
+If you
+disable `innodb_redo_log_encrypt`, any encrypted pages remain encrypted, but the
```
Comment (satya-bodapati) [inline doc/source/security/encrypting-redo-log.rst]: new line can be removed?

File: doc/source/security/encrypting-system-tablespace.rst [hunk 4]
```diff
@@ -5,8 +5,8 @@ Encrypting the System Tablespace
 ==================================================================
 
 |Percona Server| supports system tablespace encryption. The InnoDB system
-tablespace may be encrypted with the Master key encryption or may use keyring
-encryption with encryption threads. 
+tablespace may be encrypted with the master key encryption or the keyring
+encryption with advanced encryption key rotation.
```
Comment (satya-bodapati) [inline doc/source/security/encrypting-system-tablespace.rst]: either we should mention keyring encryption is a tech preview feature. Or move this to a separate line under the tech preview heading.

File: doc/source/security/encrypting-tables.rst [hunk 5]
```diff
@@ -27,13 +23,13 @@ An example of the ``CREATE TABLE`` statement:
 
 .. code-block:: mysql
 
-   mysql> CREATE TABLE myexample (id INT, mytext varchar(255)) ENCRYPTION='Y';
+   mysql> CREATE TABLE sample (id INT, mytext varchar(255)) ENCRYPTION='Y';
 
 An example of an ``ALTER TABLE`` statement.
 
-.. code-block:: MySQL
+.. code-block:: mysql
```
Comment (satya-bodapati) [inline doc/source/security/encrypting-tables.rst:30]: any difference when code-block is mysql vs text ?

**Review by satya-bodapati — CHANGES_REQUESTED** [review]
mostly looks good. minor comments

File: doc/source/security/encrypting-doublewrite-buffers.rst (hunk 2 above)
Comment (patrickbirch) [reply doc/source/security/encrypting-doublewrite-buffers.rst]: Fixed.

File: doc/source/security/encrypting-redo-log.rst (hunk 3 above)
Comment (patrickbirch) [reply doc/source/security/encrypting-redo-log.rst]: rewritten.

File: doc/source/security/encrypting-system-tablespace.rst (hunk 4 above)
Comment (patrickbirch) [reply doc/source/security/encrypting-system-tablespace.rst]: Added a line.

File: doc/source/security/encrypting-tables.rst (hunk 5 above)
Comment (patrickbirch) [reply doc/source/security/encrypting-tables.rst:30]: code-block with MySQL adds syntax coding for the code sample.

**Comment by satya-bodapati** [comment]
@patrickbirch One more thing, the commit message says vault.rst is modified. But I don't see that in the list of modified files. Maybe that line can be removed.

**Comment by patrickbirch** [comment]
Removed vault from list.

**Review by george-lorch — APPROVED** [review]

File: doc/source/security/data-masking.rst [hunk 6]
```diff
@@ -18,7 +16,7 @@ The following command installs the plugin:
 
 .. code-block:: bash
 
-    INSTALL PLUGIN data_masking SONAME 'data_masking.so';
+    shell>INSTALL PLUGIN data_masking SONAME 'data_masking.so';
```
Comment (satya-bodapati) [inline doc/source/security/data-masking.rst]: should we use mysql as code block interpreter for syntax highlighting

**Review by satya-bodapati — APPROVED** [review]
LGTM! merge after addressing review comment

File: doc/source/security/data-masking.rst (hunk 6 above)
Comment (patrickbirch) [reply doc/source/security/data-masking.rst]: Fixed.

**Pull request merged**